demos/tensorflow-image-tagging/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM korekontrol/ubuntu-java-python3
+
+RUN pip3 install --upgrade https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-1.3.0-cp34-cp34m-linux_x86_64.whl
+
+RUN apt-get install -y wget
```

Comment:
That article does not mention requiring `apt-get update` before an `apt-get install`, it mentions to combine them when `apt-get update` is used.

Reply:
In this instance I believe you're relying on the rando image not having removed it's apt-cache in order for these apt-get installs to work.  

Cleaning up the apt-cache is also detailed in the best practices document:

<pre>RUN apt-get update && apt-get install -y \
    aufs-tools \
    automake \
    build-essential \
    curl \
    dpkg-sig \
    libcap-dev \
    libsqlite3-dev \
    mercurial \
    reprepro \
    ruby1.9.1 \
    ruby1.9.1-dev \
    s3cmd=1.1.* \
 && rm -rf /var/lib/apt/lists/*</pre>

Another reason to avoid this image potentially.

Source Dockerfile: https://github.com/korekontrol/docker-ubuntu-java-python3/blob/master/Dockerfile